{
  "2f12006cc737b82b49e8941f9cafede7" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 1,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "e2c148ad2bfa6bf1",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=60056): Max retries exceeded with url: /session/757ea6add0c3c5a96bb05fc73c1249f6/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000005389220>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "time" : {
        "start" : 1594696341414,
        "stop" : 1594696348423,
        "duration" : 7009
      }
    } ]
  },
  "54e63241a4398f48d88442e979545d86" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 1,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "72f42eb159af1c78",
      "status" : "broken",
      "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=60056): Max retries exceeded with url: /session/757ea6add0c3c5a96bb05fc73c1249f6/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000003DEA910>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "time" : {
        "start" : 1594696332707,
        "stop" : 1594696339727,
        "duration" : 7020
      }
    } ]
  },
  "eae6b09ce148d4487457f782746bb0fc" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 1,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "2be14c74cfe190d9",
      "status" : "passed",
      "time" : {
        "start" : 1594696312641,
        "stop" : 1594696332677,
        "duration" : 20036
      }
    } ]
  }
}